### PR TITLE
Downgrade to Python 3.4.4 in tests

### PR DESCRIPTION
**Pull request by swenson**
And comment out `math.isclose` tests for now.

This should help address the issues in https://github.com/pybee/batavia/issues/290

### CODE REVIEW

**Comment by glasnt** [comment]
This has failed in CI, because there might be other things that need to be commented out with the minor version change.

**Comment by swenson** [comment]
Yep! Going to try again :)

File: tests/modules/test_math.py [hunk 1]
```diff
@@ -76,8 +76,8 @@ def test_constants(self):
         self.assertCodeExecution("""
             import math
             print(math.e)
-            print(math.inf)
-            print(math.nan)
+            # print(math.inf) # Not until CPython 3.5
```
Comment (freakboy3742) [inline tests/modules/test_math.py]: Is there any reason not to do the simple `if sys.version() >= (3,5)` trick here?

File: tests/modules/test_math.py [hunk 2]
```diff
@@ -207,13 +207,14 @@ def test_ldexp_int_exps_edge_cases(self):
...
-        self.assertCodeExecution("""
-            import math
-            print(math.isclose(1.0, 0.9))
-            print(math.isclose(1.0, 0.9, rel_tol=0.09))
-            print(math.isclose(1.0, 0.9, rel_tol=0.1))
-            print(math.isclose(1.0, 0.9, rel_tol=0.11))
-            print(math.isclose(1.0, 0.9, rel_tol=0.09, abs_tol=0.1))
-            print(math.isclose(1.0, 1.000000001, rel_tol=1.0, abs_tol=1.0))
-            """)
+    # not until CPython 3.5+
```
Comment (freakboy3742) [inline tests/modules/test_math.py]: This one could use the `@skipUnless(sys.version() >= (3,5))` decorator.

File: tests/modules/test_math.py (hunk 1 above)
Comment (swenson) [inline tests/modules/test_math.py]: Nope -- doing that.

File: tests/modules/test_math.py (hunk 2 above)
Comment (swenson) [inline tests/modules/test_math.py]: I like it! I did not know about this decorator.

**Comment by swenson** [comment]
I have updated this PR with the requested changes to skip instead of commenting out.

**Comment by glasnt** [comment]
Thanks @swenson! 

I'm seeing just one unexpected success in `test_dict (tests.builtins.test_tuple.BuiltinTupleFunctionTests)`. Once this is green, then it'll be a 👍 from me :)

**Comment by swenson** [comment]
Thanks @glasnt -- I think I have a fix in for that one. We'll know soon enough

**Comment by glasnt** [comment]
It's green! Are you good with this as it is now @swenson? Can it be merged?

**Comment by swenson** [comment]
I am! :)
On Oct 15, 2016 12:41 PM, Katie McLaughlin <notifications@github.com> wrote:It's green! Are you good with this as it is now @swenson? Can it be merged?

**Comment by swenson** [comment]
Thanks!
On Oct 15, 2016 1:12 PM, Katie McLaughlin <notifications@github.com> wrote:Merged #297.

**Pull request merged**